controller/src/main/java/org/jboss/as/controller/descriptions/ModelDescriptionConstants.java

```diff
@@ -120,7 +120,11 @@
     public static final String DEFAULT_REQUIRES_ADDRESSABLE = "default-requires-addressable";
     public static final String DEPLOY = "deploy";
     public static final String DEPLOYMENT = "deployment";
+    public static final String DEPLOYMENT_ADDED_NOTIFICATION = "deployment-added";
+    public static final String DEPLOYMENT_DEPLOYED_NOTIFICATION = "deployment-deployed";
     public static final String DEPLOYMENT_OVERLAY = "deployment-overlay";
+    public static final String DEPLOYMENT_REMOVED_NOTIFICATION = "deployment-removed";
+    public static final String DEPLOYMENT_UNDEPLOYED_NOTIFICATION = "deployment-undeployed";
```

Comment:
IIRC "add" and "remove" notifications should automatically be being emitted when resources are added/removed, so only deployed/undeployed is needed.